Content.Shared/Weapons/Ranged/Components/GunComponent.cs

```diff
@@ -110,6 +114,18 @@ public partial class GunComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("resetOnHandSelected")]
     public bool ResetOnHandSelected = true;
 
+    /// <summary>
+    /// Type of ammo the gun can work with
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("compatibleAmmo")]
```

Comment:
just do `[ViewVariables(VVAccess.ReadWrite), DataField]` it will set tag automatically

Reply:
Not sure what's the problem, but if I do it like this I can't load ammo into the flare gun anymore

Reply:
sus, otherwise this lgtm

Reply:
Made some more tests but yeah, it stops working. I notice that every other data field in `GunComponent.cs` is declared in this way, I don't know if there's a reason for that.

Reply:
its just old way
if the tag is the same as the field name with first letter lowercase it should work

not that important anyway